src/main/java/racingcar/model/Racing.java

```diff
@@ -0,0 +1,73 @@
+package racingcar.model;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import racingcar.exception.count.NoCountInputException;
+import racingcar.exception.count.NonNumericStringException;
+import racingcar.model.dto.CarDto;
+import racingcar.model.number.NumberGenerator;
+
+public class Racing {
+    private static final Pattern NUMBER = Pattern.compile("^\\d+$");
+    private final RacingCars racingCars;
+    private final Round round;
+
+    public Racing(RacingCars racingCars, String count) {
+        this.racingCars = racingCars;
+        this.round = createRound(count);
+    }
+
+    private Round createRound(String count) {
+        int intCount = parseIntFromString(count);
+        return new Round(intCount);
+    }
+
+    public void playRound(NumberGenerator generator) {
+        racingCars.move(generator);
+        round.nextRound();
+    }
+
+    public List<CarDto> getStatus() {
+        return racingCars.getStatus();
+    }
+
+    public List<String> getWinners() {
+        List<String> winners = new ArrayList<>();
+        for (CarDto car : racingCars.getStatus()) {
+            findWinners(car, winners);
+        }
+        return Collections.unmodifiableList(winners);
+    }
+
+    private void findWinners(CarDto car, List<String> winners) {
+        if (car.position() == racingCars.getMaxPositions()) {
+            winners.add(car.name());
+        }
+    }
+
```

Comment:
자바 16부터 Stream의 최종연산에 toList()를 사용하면 불변 객체를 반환한다고 합니다!
```suggestion
public List<String> getWinners() {
    return racingCars.getStatus().stream()
            .filter(car -> car.position() == racingCars.getMaxPositions())
            .map(CarDto::name)
            .collect(Collectors.toUnmodifiableList());
}
```

Reply:
오 엄청나게 좋은 정보 알아갑니다! 해당 내용에 대해서 좀 더 알아 보고 다음부터는 적극적으로 써보고 하겠습니다.